### DISCUSSION TITLE
How to specify the filter configuration using nginx.conf?

### DISCUSSION THREAD

**Discussion by NickUfer** [Kong/ngx_wasm_module#603] [Q&A]
How do I set a configuration for the filter I can read in through `get_vm_configuration`?

**Comment by thibaultcha** [comment]
If you are configuring the filter via nginx.conf (and not via the FFI or using Kong Gateway), then the 3rd argument to the `module` directive is the configuration: https://github.com/Kong/ngx_wasm_module/blob/main/docs/DIRECTIVES.md#module
